### ISSUE TITLE
Read GPU hot-spot temperature

### ISSUE THREAD

**Issue by fzyzcjy** [olealgoritme/gddr6#32]
Hi thank you for the code! However, in addition to memory temperature, it would be great if we could read the gpu hotspot temperature.

**Comment by Ahmed-Mahmod-Salem** [comment]
yes, this would be very useful, also packing to be one stop temp monitoring tool.

**Comment by olealgoritme** [comment]
See new branch. Only tested with RTX 4090.

**Comment by luma91** [comment]
omg this is amazing and works great on my 4090 - thankyou so much!

**Comment by olealgoritme** [comment]
Glad it's working! 🐰

**Comment by Ahmed-Mahmod-Salem** [comment]
also confirmed working on rtx 3090
